UPGRADE-API-1.13.md

```diff
@@ -63,166 +129,122 @@
 
-    `Sylius\Bundle\ApiBundle\CommandHandler\Checkout\SendShipmentConfirmationEmailHandler`:
-    ```diff
-        public function __construct(
-    -       private SenderInterface $emailSender,
-            private ShipmentRepositoryInterface $shipmentRepository,
-    +       private Sylius\Bundle\CoreBundle\Mailer\ShipmentEmailManagerInterface $shipmentEmailManager,
-        ) {
-        }
-    ```
+1. The keys for adjustment endpoints' responses have been changed from `order_item` to `orderItem` and `order_item_unit`
+   to `orderItemUnit`.
 
-    `Sylius\Bundle\ApiBundle\CommandHandler\SendContactRequestHandler`:
-    ```diff
-        public function __construct(
-    -       private SenderInterface $emailSender,
-            private ChannelRepositoryInterface $channelRepository,
-    +       private Sylius\Bundle\CoreBundle\Mailer\ContactEmailManagerInterface $contactEmailManager,
-        ) {
-        }
-    ```
+### Miscellaneous changes
 
-    `Sylius\Bundle\CoreBundle\MessageHandler\Admin\Account\SendResetPasswordEmailHandler`: 
-   ```diff
-        public function __construct(
-            private UserRepositoryInterface $shopUserRepository,
-    -       private SenderInterface $emailSender,
-    +       private Sylius\Bundle\CoreBundle\Mailer\ResetPasswordEmailManagerInterface $resetPasswordEmailManager,
-        ) {
-        }
-    ```
+1. All the `:read` serialization groups are now split to `index` and `show`.
+   By this change, the `:read` serialization group is now deprecated and will no more used in the future.
+   There is a BC layer that will allow you to use the `:read` serialization
+   group `Sylius\Bundle\ApiBundle\SerializerContextBuilder\ReadOperationContextBuilder` by adding the `read`
+   serialization group to your context.
+   Inside of this service there are 2 configurable parameters `$skipAddingReadGroup` and `$skipAddingIndexAndShowGroups`
+   that will allow you to skip adding the chosen serialization group to your context.
+   To configure skipping adding the index and show or read serialization groups to the context, add the following
+   configuration to your `config/packages/_sylius.yaml` file:
 
-1. Disabled product and taxon editing at `/admin/product-taxons/{id}` operation to improve data integrity. To modify a productTaxon, remove the existing association and create a new one.
+    ```yaml
+    sylius_api:
+        serialization_groups:
+            skip_adding_index_and_show_groups: true
+            skip_adding_read_group: true
+    ```
 
-1. The keys for adjustment endpoints' responses have been changed from `order_item` to `orderItem` and `order_item_unit` to `orderItemUnit`.
+1. Sylius serialization groups have been updated with a new prefix of `sylius:some_resource`.
+   If you extend any of the Sylius resources, you should update your serialization groups to use the new prefix.
+   Non-prefix serialization groups are deprecated and will be removed in Sylius 2.0.
```

Comment:
```suggestion
   Non-prefix serialization groups are considered deprecated and will be removed in Sylius 2.0.
```
Since we can't actually deprecate them